bigquery/requirements.txt

```diff
@@ -15,7 +15,6 @@ idna==2.7
 jsmin==2.2.2
 numpy==1.15.1
 oauthlib==2.1.0
-pandas-gbq==0.3.1
```

Comment:
Did you follow [these instructions](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery#generating-requirementstxt) to regenerate requirements.txt? I would expect more changes to this file.